README.md

```diff
@@ -467,7 +467,7 @@ Here is a collection of research papers about **Monte Carlo Tree Search**.
 }
 ```
 
-## Acknowledgments
+## Acknowledgements
```

Comment:
The words 'Acknowledgments' and 'Acknowledgements' have similar meanings of 'gratitude' or 'thanks' in English, and there is no substantial difference between the two terms except for their spelling variations. Therefore, we believe that this modification is unnecessary, but we greatly appreciate your attention to detail. Best Wishes.

Reply:
Hi @puyuan1996 
I checked now. In US English, it is `Acknowledgment` while in UK English, it is `Acknowledegement`
Since I assumed it to be a spelling error, it doesn't seem to be. 
Thanks for the wishes